assembl/static2/js/app/client.js

```diff
@@ -0,0 +1,13 @@
+import ApolloClient, { createNetworkInterface } from 'apollo-client';
+import { getDiscussionSlug } from './utils/globalFunctions';
+
+const client = new ApolloClient({
+  networkInterface: createNetworkInterface({
+    uri: `${window.location.origin}/${getDiscussionSlug()}/graphql`,
```

Comment:
I tried to use a root /graphql but couldn't with the routes order on the backend, it thoughts graphql was a discussion. We may revisit this after the routes PR is merged and when we actually need to have a graphql api for calls outside of a discussion.